alacritty/src/input.rs

```diff
@@ -565,11 +563,14 @@ impl<'a, T: EventListener, A: ActionContext<T> + 'a> Processor<'a, T, A> {
             for _ in 0..lines {
                 self.mouse_report(code, ElementState::Pressed, modifiers);
             }
-        } else if self.ctx.terminal().mode().contains(TermMode::ALT_SCREEN)
-            && faux_multiplier > 0
-            && !modifiers.shift
-        {
-            self.ctx.mouse_mut().scroll_px += new_scroll_px * faux_multiplier as i32;
+        } else if self.ctx.terminal().mode().contains(alt_screen_modes) && !modifiers.shift {
```

Comment:
~~We still need to support disabling faux scrolling with the `faux_multiplier` option for the time being.~~

After looking at the rest of the code, this is not necessary since we set the escape sequence at startup according to the `faux_multiplier`. So this comment is purely informational for other readers.